_docs/generators/generators.md

```diff
@@ -94,5 +91,8 @@ Find out how to produce Service Cutter input to calculate possible service cuts
 ![Service Cutter DDD Sample](/img/service-cutter-ddd-sample.png)
 
 ## Generic Textual Generator (Freemarker)
-With the generic generator based on [Freemarker](https://freemarker.apache.org/) templates Context Mapper users are allowed to generate arbitrary text files from CML Context Maps.
-Learn more about this generator [here](/docs/generic-freemarker-generator/)
+The generic, template-based generator allows you to generate arbitrary text files from CML Context Maps. It uses [Freemarker](https://freemarker.apache.org/) as its template engine and exposes the entire CML content as an object tree whose elements can be injected into the template.
+
+Learn more about this generator [here](/docs/generic-freemarker-generator/).
+
+<!-- TODO document new control key here? and//or on individual generator pages? -->
```

Comment:
@socadk, yes we could insert a general section at the top of the page that explains where one can find the generators in Eclipse. 

ToDo:
 * Add according section here or create issue to do it later.
 * Remove this comment afterwards.